packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue

```diff
@@ -254,36 +253,50 @@ const stratifyModel = () => {
 	kernelManager.sendMessage('reset_request', {}).register('reset_response', () => {
 		kernelManager
 			.sendMessage('stratify_request', messageContent)
-			.register('stratify_response', handleStratifyResponse)
-			.register('model_preview', handleModelPreview);
+			.register('stratify_response', (data: any) => {
+				executedCode = data.content.executed_code;
+			})
+			.register('model_preview', async (data: any) => {
+				await handleModelPreview(data);
+				saveCodeToState(executedCode, false);
+			});
 	});
 };
 
-const handleStratifyResponse = (data: any) => {
-	const executedCode = data.content.executed_code;
-	if (executedCode) {
-		codeText.value = executedCode;
-
-		// If stratify is run from the wizard, save the code but set `hasCodeBeenRun` to false
-		saveCodeToState(executedCode, false);
-	}
-};
+// const handleStratifyResponse = (data: any) => {
```

Comment:
Unsure why this is left as commented our rather than just removed
If we do not want to use `stratify_response` to `saveCodeToState` and thus the one liner is sufficient/no helper is needed can we just remove?

Are we intending to bring this back in the future? -> If we are can we leave a comment about when/why?

Reply:
removed